problemsets-mat223/linearalbegra.tex

```diff
@@ -3894,8 +3960,50 @@ \section*{Task 2.3: Pat and Jamie}
 	\]
 	\begin{parts}
 		\item How does the row space of $B$ relate to the row space of $C$?
+			\begin{solution}
+				They are equal.
+
+				Both rows of $C$ can be written as linear combinations of the 
```

Comment:
Row reduction replaces rows with linear combinations of rows. Therefore if X is the matrix Y with row operations applied, row space(X) \subseteq row space(Y). Since row reduction is reversible, in this case we have ...

Reply:
Okay, I've changed the explanation to be more along these lines. We don't have any notation for row spaces, right? Something like row(A) or rowspace(A)? 

Because without that it's kind of clunky to say "the row space of B is a subset of the row space of C", and vice versa.

Reply:
Go ahead and add a macro for `row(A)` and `col(A)`